src/main/java/core/basesyntax/model/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public interface Figure {
+    void draw();
+}
```

Comment:
figure should be abstract class and for method draw you have interface accordingly